scripts/backfill/public-project-access/backfill.py

```
@@ -0,0 +1,242 @@
+    return _post(url, body)
+
+
+def fga_request(base_url: str, path: str, body: dict) -> dict:
+    """POST a JSON request to the OpenFGA API and return the parsed response."""
+    url = f"{base_url.rstrip('/')}/{path}"
+    return _post(url, body)
+
+
+def _post(url: str, body: dict) -> dict:
+    data = json.dumps(body).encode()
+    req = urllib.request.Request(url, data=data, headers={"Content-Type": "application/json"}, method="POST")
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)